### PR TITLE
Add Getting started page

### PR DESCRIPTION
**Pull request by shaley91**

### CODE REVIEW

File: pages/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# Tools
+Most tools can be found [here]({{site.baseurl}}/pages/administrative-tooling).
+
+# Misc
+
+- [Holidays]({{site.baseurl}}/pages/holidays).
+- [Conferences and training]({{site.baseurl}}/pages/conferences-and-training).
+- [Sick Pay]({site.baseurl}/pages/sick-pay-entitlement)
+  - If your sick
```
Comment (paulanthonywilson) [inline pages/getting-started/index.md]: "you're" or "you are"

File: pages/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    - Stay at home and recover
+    - If you can work then please do
```
Comment (paulanthonywilson) [inline pages/getting-started/index.md]: "If you can work, safely and comfortably, then you may from home."

?

Comment (paulanthonywilson) [reply pages/getting-started/index.md]: (As in, if some is really feeling horrible then don't want to pressure them to be working.)

File: pages/holidays/index.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ title: Holidays
 ### How to book a holiday
 
 1. Ask your line manager's permission.
-2. Ensure that the holiday is entered into [The Holiday Spreadsheet]({{ site.baseurl }}//pages/administrative-tooling/#holiday_spread). For traceability, and for now, this will be added by the {{ md }}.
+2. Holidays are added by, err, talking to a Mexican wrestler on Slack (The Cultivar) (ask him `hols help` for all your holiday needs).
```
Comment (paulanthonywilson) [inline pages/holidays/index.md]: Holidays are added via "The Cultivar" Slack bot. Got the the `zeacultiprod` channel and type `hols help` for more information.

**Review by paulanthonywilson — CHANGES_REQUESTED** [review]

**Comment by shaley91** [comment]
Changes made.

**Review by paulanthonywilson — APPROVED** [review]

**Pull request merged**